test/e2e/diagram.navigation.zoom.pan.test.ts

```diff
@@ -22,7 +22,7 @@ import type { Point } from '@test/shared/visu/bpmn-page-utils';
 import { AvailableTestPages, PageTester } from '@test/shared/visu/bpmn-page-utils';
 import type { ImageSnapshotThresholdConfig } from './helpers/visu/image-snapshot-config';
 import { ImageSnapshotConfigurator, MultiBrowserImageSnapshotThresholds } from './helpers/visu/image-snapshot-config';
-import { ZoomType } from '@lib/component/options';
+import { ZoomType } from '../../src/component/options';
```

Comment:
**question:** is there a specific reason to change this import? TS paths were introduced recently to simplify the imports in the test code

Reply:
Sure removed it ...